IOPool/Input/test/testNoParentDictionary.sh

```diff
@@ -13,36 +13,31 @@ scram -a $SCRAM_ARCH project $CMSSW_VERSION
 pushd $CMSSW_VERSION/src
 eval `scram run -sh`
 
-# Symlink any libraries and -edmplugincache *except* for the libraries containing the test dictionaries
-for F in ${OLD_CMSSW_BASE}/lib/${SCRAM_ARCH}/* ${OLD_CMSSW_BASE}/lib/${SCRAM_ARCH}/.edmplugincache; do
-    if [ -e ${F} ] && [[ "${F}" != *"DataFormatsTestObjectsParent1"* ]]; then
-        ln -s ${F} ${CMSSW_BASE}/lib/${SCRAM_ARCH}
-    fi
-done
-
-# Symlink dictionary for edmtest::TransientIntParentT<1> from local/release
-# Also copy DataFormats/TestObjects code to be able to edit it to make ROOT header parsing to fail
-COPIED_FROM=
+# Copy DataFormats/TestObjects code to be able to edit it to make ROOT header parsing to fail
 for DIR in ${OLD_CMSSW_BASE} ${CMSSW_RELEASE_BASE} ${CMSSW_FULL_RELEASE_BASE} ; do
     if [ -d ${DIR}/src/DataFormats/TestObjects ]; then
         mkdir DataFormats
         cp -r ${DIR}/src/DataFormats/TestObjects DataFormats/
-        cp ${DIR}/lib/${SCRAM_ARCH}/*DataFormatsTestObjectsParent1* ${CMSSW_BASE}/lib/${SCRAM_ARCH}
-        COPIED_FROM=${DIR}
         break
     fi
 done
-if [ ! -e ${CMSSW_BASE}/lib/${SCRAM_ARCH}/libDataFormatsTestObjectsParent1.so ]; then
-    echo "Failed to symlink libDataFormatsTestObjectsParent1.so from local or release area"
-    exit 1;
-fi
 if [ ! -e DataFormats/TestObjects ]; then
     echo "Failed to copy DataFormats/TestObjects from local or release area"
     exit 1;
 fi
 
+# Enable TransientIntParentT dictionaries
+cat DataFormats/TestObjects/test/BuildFile_extra.xml >> DataFormats/TestObjects/test/BuildFile.xml
+scram build -j $(nproc)
+
 popd
 
+# Add OLD_CMSSW_BASE in between CMSSW_BASE and CMSSW_RELEASE_BASE for
+# LD_LIBRARY_PATH and ROOT_INCLUDE_PATH
+LD_LIBRARY_PATH_TO_OLD=${OLD_CMSSW_BASE}/biglib/${SCRAM_ARCH}:${OLD_CMSSW_BASE}/lib/${SCRAM_ARCH}:${OLD_CMSSW_BASE}/external/${SCRAM_ARCH}/lib
```

Comment:
I would suggest to remove this line and add something like the following after line 12 above
```
LD_LIBRARY_PATH_TO_OLD=$(echo $LD_LIBRARY_PATH | tr ':' '\n' | grep "^${CMSSW_BASE}/" | tr '\n' ':')
```

Reply:
Thanks, done.